packages/x-data-grid/src/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -456,6 +456,7 @@ export const useGridColumnResize = (
 
   const storeReferences = (colDef: GridStateColDef, separator: HTMLElement, xStart: number) => {
     const root = apiRef.current.rootElementRef.current!;
+    const escapedColDefField = CSS.escape(colDef.field);
```

Comment:
The jsdom test is failing because `CSS.escape` doesn't exist in that context. After discussion, we have decided that we want to preserve the bug in jsdom. Could you avoid `CSS.escape` if it is not available? Something like `const escapedColDefField = typeof CSS === 'undefined' ? colDef.field : CSS.escape(colDef.field);`. Also add a comment that explains the hack.

cc @cherniavskii

Reply:
Sure, thanks for the suggestion.